euth/organisations/models.py

```diff
@@ -0,0 +1,28 @@
+from django.conf import settings
+from django.db import models
+from model_utils import models as model_utils
+
+
+class OrganisationManager(models.Manager):
+    def get_by_natural_key(self, name):
+        return self.get(name=name)
+
+
+class Organisation(model_utils.TimeStampedModel):
+    name = models.CharField(max_length=512, unique=True)
+    slug = models.SlugField(max_length=512, unique=True)
+    description_why = models.TextField()
+    description_how = models.TextField()
+    description = models.TextField()
```

Comment:
No, we are planning to use [django-parler](https://github.com/django-parler/django-parler) for a different project. The main difference is how the translations are stored: django-modeltranslation adds fields for each language to the model while django-parler uses a separate table.